Kernel_23/test/Kernel_23/include/CGAL/_test_io.h

```diff
@@ -40,7 +40,7 @@ _test_io_for(const T& t)
     }
 
     std::ifstream iFile(TEST_FILENAME, std::ios::in);
-    T u;
+    T u = t;
```

Comment:
This one?

Reply:
I am not sure that we can modify sufficiently the `operator>>` for all kernel objects to convince the compiler that all is right. As the warning was in a test, I would say it is sufficient for me to quiet the warning.

Possible issue: the warning may resurface in any user-code.